ninja-servlet/src/main/java/ninja/servlet/file/NinjaFileItemStream.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2012-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ninja.servlet.file;
+
+import org.apache.commons.fileupload.FileItemStream;
+
+public interface NinjaFileItemStream extends FileItemStream {
```

Comment:
Is it really necessary to extend FileItemStream? Wouldn't it be sufficient to just add outputStream (and maybe some other methods) to the interface?